src/middleware.ts

```diff
@@ -187,3 +188,19 @@ export const withJWTVerified =
 
     return handler(request);
   };
+
+export const withExistingRegistration =
```

Comment:
1. Can we work on the naming better? I'm not sure if with is clear
2. Can we have some folder `middleware/` and there 1 file per middleware + test + index that exports them?

Reply:
1. Changed to a more explicit name, is it better?
2. Okay! Can it be a separate PR imminently after this one? In this one, I've followed the existing structure and would like to pollute APL changes with middleware refactor

Reply:
Issue for middleware [#29](https://github.com/saleor/saleor-app-sdk/issues/29)